src/__init__.py

```diff
@@ -364,7 +365,7 @@
                           'Please upgrade your simuPOP installation.')
     if 'DBG_COMPATIBILITY' in simuOptions['Debug']:
         print(
-            'WARNING: parameter revision in simuOpt.setOptions is obsolete and might be removed from a future version of simuPOP.'
+            'WARNING: parameter revision in simuOpt.setOptions is obsolete and might be removed from a future version of simuPOP.', file=sys.stderr
```

Comment:
@BoPeng I inlined the function per your request. I built the docker locally and verified banner is printed to stderr and not to stdout. BTW, building the Docker locally takes a considerable amount of time on my local MacOS. I closed the previous PR and opened a new one as I didn't quiet get the previous PR to "refresh" with my code.